app/objects/c_operation.py

```diff
@@ -258,9 +260,19 @@ async def run(self, services):
             while not await self.is_closeable():
                 await asyncio.sleep(10)
             await self.close(services)
+
+            # Automatic event log output
+            file_svc = services.get('file_svc')
+            await self.write_event_logs_to_disk(file_svc, data_svc, output=True)
         except Exception as e:
             logging.error(e, exc_info=True)
 
+    async def write_event_logs_to_disk(self, file_svc, data_svc, output=False):
```

Comment:
can you make a small adjustment here to specific the event logs written to disk. operation_event_logs etc.. this would differentiate from a function that writes all events to disk